backend/packages/Upgrade/src/api/controllers/validators/FeatureFlagValidator.ts

```diff
@@ -1,5 +1,15 @@
 import { Type } from 'class-transformer';
-import { IsNotEmpty, IsDefined, IsOptional, ValidateNested, IsString, IsBoolean, IsArray } from 'class-validator';
+import {
+  IsNotEmpty,
+  IsDefined,
+  IsOptional,
+  ValidateNested,
+  IsString,
+  IsBoolean,
+  IsArray,
+  IsEnum,
+} from 'class-validator';
+import { FEATURE_FLAG_STATUS } from 'upgrade_types';
 
 class FlagVariationValidator {
```

Comment:
FeatureFlag Validators should now contain `context` array validations

Reply:
I can make that change, but I thought we could leave the API changes that are not necessary for passing the CI to the API tickets: https://github.com/CarnegieLearningWeb/UpGrade/issues/1345 and https://github.com/CarnegieLearningWeb/UpGrade/issues/1307 

There will be many more changes necessary to complete those.